### ISSUE TITLE
2.5.1.1 and rubocop problem

### ISSUE THREAD

**Issue by enzinia** [whitequark/parser#519]
With `parser-2.5.1.1` and `rubocop` I get many false offences like 
```
Layout/SpaceBeforeFirstArg: Put one space between the method name and the first argument.
class ApplicationRecord < ActiveRecord::Base
```
or
```
Layout/IndentationWidth: Use 2 (not -31) spaces for indentation.
    authorize Task, :update?
```

When fallback to parser-2.5.1.0, all became to work normally. Not sure that it is `rubocop` problem, since before 2.5.1.1 it worked well.

Log to demonstrate behavior:

```
$ docker run -it --rm -v `pwd`:/app ruby:2.5.1-alpine sh
/ # cd app/
/app # gem install rubocop -v 0.56.0
Fetching: parallel-1.12.1.gem (100%)
Successfully installed parallel-1.12.1
Fetching: ast-2.4.0.gem (100%)
Successfully installed ast-2.4.0
Fetching: parser-2.5.1.1.gem (100%)
Successfully installed parser-2.5.1.1
Fetching: powerpack-0.1.2.gem (100%)
Successfully installed powerpack-0.1.2
Fetching: rainbow-3.0.0.gem (100%)
Successfully installed rainbow-3.0.0
Fetching: ruby-progressbar-1.9.0.gem (100%)
Successfully installed ruby-progressbar-1.9.0
Fetching: unicode-display_width-1.4.0.gem (100%)
Successfully installed unicode-display_width-1.4.0
Fetching: rubocop-0.56.0.gem (100%)
Successfully installed rubocop-0.56.0
8 gems installed
/app #
/app # rubocop
Inspecting 58 files
........CCW.CWWWCC...........CCCCCCCCCCCWWWWWWCCCCCCCCCCC.

Offenses:

app/models/application_record.rb:1:39: C: Layout/SpaceBeforeFirstArg: Put one space between the method name and the first argument.
class ApplicationRecord < ActiveRecord::Base
                                      
app/mailers/application_mailer.rb:1:39: C: Layout/SpaceBeforeFirstArg: Put one space between the method name and the first argument.
class ApplicationMailer < ActionMailer::Base
                                      
*_controller.rb:3:5: C: Layout/IndentationWidth: Use 2 (not -31) spaces for indentation.
    authorize Task, :update? ...
    ^^^^^^^^^^^^^^^^^^^^^^^^
*_controller.rb:6:3: W: Layout/DefEndAlignment: end at 6, 2 is not aligned with ApplicationController
/app # 
/app # gem uninstall parser -ax --force
Removing ruby-parse
Removing ruby-rewrite
Successfully uninstalled parser-2.5.1.1
/app # gem install parser -v 2.5.1.0
Fetching: parser-2.5.1.0.gem (100%)
Successfully installed parser-2.5.1.0
1 gem installed
/app # rubocop
Inspecting 58 files
..........................................................

58 files inspected, no offenses detected
```

**Comment by wongy91** [comment]
same here ` E: Lint/Syntax: unexpected token kUNLESS_MOD` and `E: Lint/Syntax: unexpected token kEND`

**Comment by jodosha** [comment]
Same problem here.

**Comment by wongy91** [comment]
downgrading the gem works, im using `2.5.0.5`

**Comment by bbatsov** [comment]
I’ve patched RuboCop to ignore 2.5.1.1 and issued 0.58.1.

**Comment by whitequark** [comment]
Dupe of #518.
